ReportUtilityBills/measure.rb

```diff
@@ -203,21 +203,15 @@ def energyPlusOutputRequests(runner, user_arguments)
 
     # Has production
     has_pv = @hpxml_buildings.count { |hpxml_bldg| !hpxml_bldg.pv_systems.empty? } > 0
-    has_battery = @model.getElectricLoadCenterStorageLiIonNMCBatterys.size > 0 # has modeled battery
-    has_generator = @hpxml_buildings.count { |hpxml_bldg| !hpxml_bldg.generators.empty? } > 0
 
     # Fuel outputs
     fuels.each do |(fuel_type, is_production), fuel|
-      fuel.meters.each do |meter|
-        next unless has_fuel[hpxml_fuel_map[fuel_type]]
-        next if is_production && !has_pv # we don't need to request these meters if there isn't pv
-        next if meter.include?('ElectricStorage') && !has_battery # we don't need to request this meter if there isn't a modeled battery
-        next if meter.include?('Cogeneration') && !has_generator # we don't need to request this meter if there isn't a generator
-
-        result << OpenStudio::IdfObject.load("Output:Meter,#{meter},monthly;").get
-        if fuel_type == FT::Elec && @hpxml_header.utility_bill_scenarios.has_detailed_electric_rates
-          result << OpenStudio::IdfObject.load("Output:Meter,#{meter},hourly;").get
-        end
+      next unless has_fuel[hpxml_fuel_map[fuel_type]]
+      next if is_production && !has_pv # we don't need to request this meter if there isn't pv
```

Comment:
... otherwise we get a warning? an error?

Reply:
Just tested it. You get an EnergyPlus warning:
```
** Warning ** Output:Meter: invalid Key Name="ELECTRICITY:PV" - not found.
```